lib/reply.js

```diff
@@ -445,16 +440,9 @@ function onSendEnd (reply, payload) {
 
   // we check if we need to update the trailers header and set it
   if (reply[kReplyTrailers] !== null) {
-    const trailerHeaders = Object.keys(reply[kReplyTrailers])
-    let header = ''
-    for (const trailerName of trailerHeaders) {
-      if (typeof reply[kReplyTrailers][trailerName] !== 'function') continue
-      header += ' '
-      header += trailerName
-    }
     // it must be chunked for trailer to work
     reply.header('Transfer-Encoding', 'chunked')
-    reply.header('Trailer', header.trim())
```

Comment:
I think we can completely delete `filter` here (and also in `sendTrailer()`) because it's already validated in `Reply.prototype.trailer()`, but if we leave it, then we must also check it for invalid trailers (`INVALID_TRAILERS`), wdyt?

Reply:
No, `removeTrailer` will set the value to `undefined`. It is required to check if the value is function.